assets/app/view/game/round/auction.rb

```diff
@@ -181,8 +181,15 @@ def render_company_actions(company)
 
           buttons = []
           if @step.may_bid?(company) && @step.min_bid(company) <= @step.max_place_bid(@current_entity, company)
-            bid_str = @step.respond_to?(:bid_str) ? @step.bid_str(company) : 'Place Bid'
-            buttons << h(:button, { on: { click: -> { create_bid(company, input) } } }, bid_str)
+            if @step.respond_to?(:bid_choices) && (choices = @step.bid_choices(company))
```

Comment:
I see 18FL's auction step has `bid_choices` defined, does it still render properly?

https://github.com/tobymao/18xx/blob/master/lib/engine/game/g_18_fl/step/buy_cert.rb#L16

Reply:
18FL won't have any situation where this code path would be reached, none of its steps define `may_bid?` or inherit from `Step::Auctioner`.

18FL uses `bid_choices` for a round of blind bids at the start of the game, to determine player order when selecting private companies; `bid_choices` is called from `render_turn_bid`.

That said, it might not have been the best choice to reuse this name. If any future games want have both 18Ardennes's assigned bids and 18FL's blind bids then there will be a conflict. Should I rename this method?

Reply:
I think it's fine, don't need to future-proof for hypotheticals